### PR TITLE
Update APIs.md

### PR DESCRIPTION
**Pull request by DoviMaj**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/master/CONTRIBUTING.md).

#### 1.Describe the changes made:

Grammar revision

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#XXXXX

### CODE REVIEW

File: nodeJS/APIs/APIs.md [hunk 1]
```diff
@@ -27,26 +27,26 @@ _However_, REST (an acronym for Representational State Transfer) is a popular an
...
 
 Setting up CORS in express is very easy, there's a middleware that does the work for us. [This article](https://medium.com/@alexishevia/using-cors-in-express-cac7e29b005b) explains everything you need to do, and the official docs can be found [here](https://expressjs.com/en/resources/middleware/cors.html).
 
-For now it is acceptable to just allow access from any origin.. this makes development quite a bit easier but for any _real_ project, once you deploy to a production environment you will probably want to specifically block access from any origin _except_ your front-end website. The documentation above explains how to do this.
+For now, it is acceptable to just allow access from any origin.. this makes development quite a bit easier but for any _real_ project, once you deploy to a production environment you will probably want to specifically block access from any origin _except_ your front-end website. The documentation above explains how to do this.
```
Comment (twalton83) [inline nodeJS/APIs/APIs.md:44]: "For now, it is acceptable to just allow access from any origin.." <- let's get rid of the extra period here.

**Review by twalton83 — COMMENTED** [review]
Caught one small grammar issue, otherwise looks good.

File: nodeJS/APIs/APIs.md (hunk 1 above)
Comment (DoviMaj) [reply nodeJS/APIs/APIs.md]: done

Comment (DoviMaj) [reply nodeJS/APIs/APIs.md]: ops misread it. let me remove the period

**Review by twalton83 — APPROVED** [review]
Thanks for updating.

**Pull request merged**